plugins/github-actions/src/components/WorkflowRunDetails/useWorkflowRunJobs.ts

```diff
@@ -37,7 +37,7 @@ export const useWorkflowRunJobs = ({
           repo,
           id: parseInt(id, 10),
         })
-      : Promise.reject('No repo/owner provided');
+      : Promise.reject(Error('No repo/owner provided'));
```

Comment:
Preferred pattern is to use `new Error`, although I think the linter will complain about that too x)

Reply:
Done!